### ISSUE TITLE
Make modals accessible for screen readers

### ISSUE THREAD

**Issue by klues** [asterics/AsTeRICS-Grid#210]
Accessibility of modals for screen readers currently is very limited. The modal for "set navigation" in edit mode has better accessibility, which also should be added to other modals, at least:
* set correct properties `aria-modal="true"` and `role="dialog"` , see [setNavigationModal.vue:5](https://github.com/asterics/AsTeRICS-Grid/blob/master/src/vue-components/modals/setNavigationModal.vue#L5)
* set focus within the modal, using `v-focus` or `.focus()`, either on an input element or on the "X" close link at the top, see [setNavigationModal.vue:16](https://github.com/asterics/AsTeRICS-Grid/blob/master/src/vue-components/modals/setNavigationModal.vue#L16)

This change shouldn't be done on each modal manually, but a new Vue component like `baseModal.vue` should be created which takes care of all basic things for modals and is used by all other modals.

**Comment by klues** [comment]
Since there is now the new HTML5 element `dialog`, this should be used instead of the `aria` and `role` properties, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog
